server/chat-plugins/announcements.ts

```diff
@@ -70,8 +122,9 @@ export const commands: ChatCommands = {
 
 			const source = supportHTML ? this.checkHTML(target) : Utils.escapeHTML(target);
 
-			room.minorActivity = new Announcement(room, source);
+			room.minorActivity = new Announcement(room, {source, activityId: 'announcement'});
```

Comment:
`activityId` shouldn't be specified here; like I said, they should be two different types that extend from each other.